src/chainer.js

```diff
@@ -12,7 +13,7 @@ function chainer(functions) {
     }
 
     return result;
-  }
+  };
```

Comment:
Unnecessary semicolon at the end of function block. Please remove it to keep the code clean and consistent.